explainer.md

```diff
@@ -0,0 +1,107 @@
+# Screen Wake Lock API
+
+## Introduction
+The **Screen Wake Lock API** allows web applications to prevent a device's
+screen from dimming and locking.
+
+See also:
+* [The current specification](https://w3c.github.io/screen-wake-lock/)
+* [web.dev article](https://web.dev/wakelock/) on the Screen Wake Lock API
+
+## Why?
+To avoid draining the battery, most devices quickly go to sleep when left idle.
+While this is fine most of the time, some applications need to keep the screen
+awake to complete their work. For example:
+
+* A recipe app that keeps the screen on while you bake a cake or cook dinner.
+* A boarding pass or ticket app that keeps the screen on until the barcode has
+  been scanned.
+* A kiosk-style app that keeps the screen on continuously.
+* A web-based presentation app that keeps the screen on during a presentation.
```

Comment:
This one is ok to keep, but calls into question wether we need some kind of integration with the Fullscreen API. 

For example:

```JS
body.requestFullscreen({wakelock: "screen"})
```

Reply:
Or also the [Presentation API](https://w3c.github.io/presentation-api/)?

Reply:
As I understand it the current user agent behavior of holding a wake lock while a <video> tag is playing is unspecified. From a developer ergonomics perspective seems reasonable to add well-specified wake lock integrations to APIs where a wake lock may be frequently requested but I'll push back on the implication that a generic screen wake lock API is unnecessary because of those integrations.

Reply:
I think both API options might be nice. This API could handle the "cooking site" use case, while Presentation and Fullscreen API could handle the "presentation" use case. I filed https://github.com/whatwg/fullscreen/issues/185